src/components/sign-up/sign-up.component.jsx

```diff
@@ -2,57 +2,61 @@ import React from 'react';
 import FormInput from '../form-input/form-input.component';
 import axios from 'axios';
 import './sign-up.styles.scss';
+import { URL } from '../../env'
+import { signup } from '../../env'
+import { setItem } from '../../utils'
+import { removeItem } from '../../utils'
+import { getItem } from '../../utils'
 
 class SignUp extends React.Component {
   constructor() {
     super();
 
     this.state = {
       username: '',
-      fname: '',
-      lname: '',
+      first_name: '',
+      last_name: '',
       email: '',
       password: '',
     };
   }
 
   handleSubmit = event => {
     event.preventDefault();
-    const { username, fname, lname, email, password } = this.state;
+    const { username, first_name, last_name, email, password } = this.state;
     const user = {
       username: this.state.username,
-      fname: this.state.fname,
-      lname:this.state.lname,
-      email:this.state.email,
+      first_name: this.state.first_name,
+      last_name: this.state.last_name,
+      email: this.state.email,
       password: this.state.password
     };
     console.log(user);
 
-      axios.post('https://soud-cloud-backend.herokuapp.com/api/user/create/', user)
+    axios.post(`${URL}${signup}`, user)
```

Comment:
Same as above. Try creating a generic function for GET and POST and use that throughout